app/src/features/inntektsmelding/Steg1DineOpplysninger.tsx

```diff
@@ -24,17 +25,16 @@ import {
   formatYtelsesnavn,
   slåSammenTilFulltNavn,
 } from "~/utils";
-import { useOpplysninger } from "~/views/ny-inntektsmelding/OpplysningerContext";
 
 import { Informasjonsseksjon } from "../Informasjonsseksjon";
+import { Fremgangsindikator } from "../skjema-moduler/Fremgangsindikator";
 import { useDocumentTitle } from "../useDocumentTitle";
-import { Fremgangsindikator } from "./Fremgangsindikator";
 
 type PersonOgSelskapsInformasjonForm = NonNullable<
   InntektsmeldingSkjemaState["kontaktperson"]
 >;
 
-export const PersonOgSelskapsInformasjonSeksjon = () => {
+export const Steg1DineOpplysninger = () => {
```

Comment:
Jeg endret navngivningen til å være lik med det jeg gjorde for refusjon. Synes det var enklere å forstå.